pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -962,26 +967,10 @@ func TestHighProirotyFlushUnschedulableQLeftover(t *testing.T) {
 		},
 	}
 
-	addOrUpdateUnschedulablePod(q, &highPod)
-	addOrUpdateUnschedulablePod(q, &midPod)
-
-	// Update pod condition to highPod.
-	podutil.UpdatePodCondition(&highPod.Status, &v1.PodCondition{
```

Comment:
Please keep these status updates and move them above the lines that add pods to the unschedulableQ.

Reply:
Done thank you